### ISSUE TITLE
Drawing mode buttons are overflowing in landscape mode

### ISSUE THREAD

**Issue by Zverik** [Zverik/every_door#715]
subj.

**Comment by pkoby** [comment]
I'm really excited for drawing mode, so I decided to download the build and try it out.

I think this is the same issue as your title: on my small phone (Moto x4), the bottom three buttons of drawing mode overlap the bottom bar (but are still on the screen), and they are nearly unselectable (top 5% of the button works). It's very tough to lock it or change to erasing.

Based on how things line up, it looks like anything below the bottom of the blue + button is untappable.
![Screenshot_20240428-123415](https://github.com/Zverik/every_door/assets/7786285/a576e0c0-6f3b-4cc1-bf2c-70f36fb8d8c1)

**Comment by Zverik** [comment]
Hey @pkoby, thanks for testing! Did you download the build I made today, at https://github.com/Zverik/every_door/actions/runs/8867937078 ? It should say "5.0-beta1" on loading and on the about page. if not, please test it again :)

**Comment by pkoby** [comment]
Yep, that's the one I got on my phone. I downloaded the previous one to my small tablet earlier today.

**Comment by Zverik** [comment]
@pkoby could you please test this build? https://github.com/Zverik/every_door/actions/runs/8869095873

After you've seen the same glitch, please go to settings -> about → system log and copy me the line with StyleChooserButton (or send me it with the send button).

**Comment by pkoby** [comment]
I downloaded it, and it's working great in portrait mode now. Turning to landscape on my phone is still a mess, though. I went to the system log (had to switch modes to get a menu button, though, which probably should be added to the note mode?), and there wasn't anything in there. I sent it to you with the send button, so maybe there's something I couldn't see.

**Comment by Zverik** [comment]
Yeah for landscape mode there's not enough space and I've no idea for now how to rearrange buttons. Could you maybe post a screenshot so I have something to think off?

**Comment by pkoby** [comment]
Oops, I meant to do that already... 
![Screenshot_20240428-131657](https://github.com/Zverik/every_door/assets/7786285/053db567-542c-4950-b81d-89ecd193000f)
